chatwoot/chatwoot
# Mount local storage to host

## Description

Mount local storage to host by default, to avoid data loss when self-hosted by docker.

Fixes #1166 #1258

## Type of change


## How Has This Been Tested?

- Deploy Chatwoot by docker using default docker-compose config with local storage
- Update the channel avatar of an inbox
- Redeploy Chatwoot
- Check whether the channel avatar preserved

## Checklist:
